README.md

```diff
@@ -7,7 +7,7 @@ In addition to the plugin code, this repository contains additional files for th
 
 ## Installation of the DDDC Plugin
 
-As the DDDC plugin is a standard Netbox plugin, it can be installed according to the [Netbox documentation](https://docs.netbox.dev/en/stable/plugins/#installing-plugins). This plugin is compatible with Netbox version 4.1.x and ensured by plugins `setup.py` file.  
+As the DDDC plugin is a standard Netbox plugin, it can be installed according to the [Netbox documentation](https://docs.netbox.dev/en/stable/plugins/#installing-plugins). This plugin is compatible with Netbox version 4.1.x and ensured the `Dockerfile` file.
```

Comment:
The second sentence does not make sense: What does "and ensured the `Dockerfile` file" mean? Maybe, you are missing the word "by"? (Not quite sure, how a docker file would ensure that but that sounds most likely.)
_Pro tip: Use one line per sentence ;-)_
```suggestion
As the DDDC plugin is a standard Netbox plugin, it can be installed according to the [Netbox documentation](https://docs.netbox.dev/en/stable/plugins/#installing-plugins). 
This plugin is compatible with Netbox version 4.1.x and ensured by the docker file.